client/src/components/Account/Account.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+import Login from './Login.js';
+import { Container } from 'react-bootstrap';
+import { AccountContext } from './AccountContext.js';
+import Register from './Register.js';
+
+const Account = () => {
+    const [activeWindow, setActiveWindow] = useState("signin");
+
+    const switchToSignup = () => {
+        setActiveWindow("signup")
+    }
+
+    const switchToSignin = () => {
+        setActiveWindow("signin")
+    }
+
+    const contextValue = {switchToSignup, switchToSignin};
+    return (
+        <AccountContext.Provider value={contextValue}>
+        <Container className="border rounded">
+
+            {activeWindow === "signin" && <Login />}
+            {activeWindow === "signup" && <Register />}
+
+        </Container>
+        </AccountContext.Provider>
+    );
+}
+
+export default Account;
```

Comment:
EOF line

Reply:
Nice job! I wonder if this can get put into a Modal though, specifically the one with the grid like this - https://react-bootstrap.github.io/components/modal/ 

lemme know what you think!

Reply:
Haven't seen these before but I think it'll look better than what I have now. I'll look into setting one up